packages/components/psammead-podcast-promo/README.md

```diff
@@ -53,7 +53,7 @@ This component is a basic `p` that is styled to be used for the synopsis/descrip
 
 ### Components: `PodcastPromo.Card.CallToAction`
```

Comment:
the name of this needs updating?

Reply:
Ah yes - I had but hadn't pushed the commit. It's updated now